src/test/run-make/i128-ffi/test.c

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Just use the rust_test_helpers.c file that we have precisely for the C FFI testing.